Mergin/ui/ui_sync_direction_page.ui

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>WizardPage</class>
+ <widget class="QWizardPage" name="WizardPage">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>400</width>
+    <height>174</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>WizardPage</string>
+  </property>
+  <layout class="QVBoxLayout" name="verticalLayout">
+   <item>
+    <widget class="QLabel" name="label">
+     <property name="text">
```

Comment:
Can you add a section in the first wizard page with link to https://merginmaps.com/docs/dev/dbsync/
and a short description of tool like

"Mergin Maps DB Sync is a tool that takes care of two-way synchronisation between Mergin Maps and PostGIS databases. Read the description on URL"

Reply:
Done
![image](https://github.com/MerginMaps/qgis-mergin-plugin/assets/776954/a4d6ded3-52ac-4c7a-b3a5-c6ce489f5231)